docs/guide/docs/tutorials/uipath.md

```diff
@@ -0,0 +1,183 @@
+   * @param serviceLogicalName The UiPath Service Logical Name
+   * @param accessToken The UiPath Access Token. Use the getBotpressToken() to generate a UiPath Access Token.
+   * @param processKey The UiPath Process Key. For example, "MyProcess".
+   * @param processKey The UiPath Process Version. For example, "1.10.13".
+   * @return The Release Key for the given UiPath Process
+   */
+  const getReleaseKey = async (accountLogicalName, serviceLogicalName, accessToken, processKey, processVersion) => {
+    const resp = await axios.get(
+      `https://platform.uipath.com/${accountLogicalName}/${serviceLogicalName}/odata/Releases`,
+      {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+          'X-UIPATH-TenantName': serviceLogicalName
+        }
+      }
+    )
+
+    const release = resp.data.value.find(
+      release => release.ProcessKey === processKey && release.ProcessVersion === processVersion
+    )
+    return release.Key
+  }
+
+  /**
+   * Starts a UiPath Job using the UiPath Orchestrator
+   * @title Start UiPath Job
+   * @category UiPath
+   * @author Botpress
+   */
+  const myAction = async () => {
+    const clientId = '6v7PmPJL6FOGu6RB8I1Y4adLXhIwovQN' // PLACHOLDER. Replace with your own Client ID
+    const refreshToken = 'N1GNVn4sIqWz4wDMTLUlTdXyMZtYmpBZG_7zN5IYcXhC6' // PLACHOLDER. Replace with your own Refresh Token
+    const accountLogicalName = 'botprgaoyxrd' // PLACHOLDER. Replace with your own Account Logical Name
+    const serviceLogicalName = 'BotpressDefmqn8231188' // PLACHOLDER. Replace with your own Service Logical Name
+    const processKey = 'MyProcess' // PLACHOLDER. Replace with your own Process Key
+    const processVersion = '1.0.18' // PLACHOLDER. Replace with your own Process Version
+
+    const accessToken = await getAccessToken(clientId, refreshToken)
+    const botpressToken = await getBotpressToken()
+    const releaseKey = await getReleaseKey(
+      accountLogicalName,
+      serviceLogicalName,
+      accessToken,
+      processKey,
+      processVersion
+    )
+
+    const { data, status } = await axios.post(
+      `https://platform.uipath.com/${accountLogicalName}/${serviceLogicalName}/odata/Jobs/UiPath.Server.Configuration.OData.StartJobs`,
+      JSON.stringify({
+        startInfo: {
+          ReleaseKey: releaseKey,
+          Strategy: 'All',
+          RobotIds: [],
+          NoOfRobots: 0,
+          InputArguments: JSON.stringify({
+            channel: event.channel,
+            target: event.target,
+            botId: event.botId,
+            botpressToken
+          })
+        }
+      }),
+      {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+          'Content-Type': 'application/json',
+          'X-UIPATH-TenantName': serviceLogicalName
+        }
+      }
+    )
+
+    if (status !== 201) {
+      throw `Error from UiPath (${status})`
+      console.log(data)
```

Comment:
This console log will not be called since you throw before. Also, in actions I suggest using bp.logger.error so they are broadcasted to the logging panel on the studio.